server/integration-tests/calabash-test-suite/features/steps/shared.rb

```diff
@@ -14,8 +14,7 @@
 
 Then(/^I print it to stdout$/) do
   s = query("edittext", :getText).first.gsub("URI: //ready\nparams: {json={}\n}", "").gsub("URI: //ping\nparams: {json={}\n}", "")
-  puts s
-  $stdout.puts s
+  Kernel.puts s
```

Comment:
What the difference between all these three calls? Why have you changed the previous approach?

Reply:
@leffsu said that it can be safely reverted.